tests/test_ble_interface_fixtures.py

```
@@ -246,6 +246,14 @@ async def write_gatt_char(self, *_args: Any, **_kwargs: Any) -> None:
             """Accept any positional and keyword arguments and perform no action."""
             return None
 
+        async def pair(self, **_kwargs: Any) -> None:
```

Ruff (0.15.2)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `**_kwargs`

(ANN401)